### PR TITLE
Add CloudSQL flag

### PR DESCRIPTION
**Pull request by bkniffler**
`--add-cloudsql-instances` as documented in https://cloud.google.com/sql/docs/mysql/connect-run#node.js

### CODE REVIEW

**Comment by bkniffler** [comment]
@googlebot I signed it!

File: src/deploy-cloudrun.ts [hunk 1]
```diff
@@ -134,6 +135,10 @@ export async function run(): Promise<void> {
         cmd.push('--update-secrets', secrets);
         installBeta = true;
       }
+      if (cloudsql) {
+        cmd.push('--add-cloudsql-instances', cloudsql);
+        installBeta = true;
```
Comment (averikitsch) [inline src/deploy-cloudrun.ts:140]: installBeta is unnecessary

File: src/deploy-cloudrun.ts [hunk 2]
```diff
@@ -134,6 +135,10 @@ export async function run(): Promise<void> {
         cmd.push('--update-secrets', secrets);
         installBeta = true;
       }
+      if (cloudsql) {
```
Comment (averikitsch) [inline src/deploy-cloudrun.ts:138]: Add to line 100 that it will be ignored with metadata

**Review by averikitsch — CHANGES_REQUESTED** [review]

**Review by averikitsch — COMMENTED** [review]
I apologize, I forgot to mention that you should remove the generated dist/index.js from the PR. This will be generated on merge.

**Comment by JeandeCampredon** [comment]
Do you plan at some point to merge this ?

**Review by averikitsch — APPROVED** [review]

**Comment by averikitsch** [comment]
Thank you for your patience. Can you resolve the merge conflict?

**Comment by nadavzipo** [comment]
So this should be handled in with the cli/console at the moment?
Waiting as well.

**Comment by faizeev47** [comment]
Any update on this?

**Comment by FaizeKhan12** [comment]
Any update when this is coming in **google APIS SDK?**

**Comment by 178inaba** [comment]
@bkniffler @averikitsch 
hello.
I tried to resolve the conflict in this pull request, but the configuration has changed, such as src/deploy-cloudrun.ts being deleted, so the conflict could not be resolved.
So I recreated a new pull request.
https://github.com/google-github-actions/deploy-cloudrun/pull/389

**Comment by vincejv** [comment]
To anyone that can't wait for this PR to be merged, you can use the `flags` input to use the arguments that are currently not supported

```
flags:
    description: |-
      Space separated list of other Cloud Run flags, examples can be found:
      https://cloud.google.com/sdk/gcloud/reference/run/deploy#FLAGS. Ex
      --service-account=my-account@project.iam.gserviceaccount.com --concurrency=10
    required: false
```

Sample step would be for customizing the CPU and Memory during deployment
```
  - name: Deploy to Cloud Run
    id: deploy
    uses: google-github-actions/deploy-cloudrun@v0
    with:
      service: ${{ env.SERVICE }}
      region: ${{ env.REGION }}
      image: ${{ env.REGISTRY }}/${{ env.PROJECT_ID }}/${{ env.GHUB_REPO_NAME }}/${{ env.SERVICE }}:${{ github.sha }}
      project_id: ${{ env.PROJECT_ID }}
      flags: --cpu 1000m --memory 384Mi
```

Cheers!

**Comment by verbanicm** [comment]
Closed as duplicate of #389

**Pull request closed without merging**